js/intro.js

```diff
@@ -256,8 +260,7 @@ async function showSavedDetails() {
     }
     const usersRequest = await makeApiCall(`${BASE_URL}/users/${userId}/intro`);
     if (usersRequest.status === 200) {
-      const usersDataList = await usersRequest.json();
-      const userData = usersDataList.data[0];
+      const userData = usersRequest.data.data[0];
```

Comment:
`Suggestion:` You can use data like this.

```js
const userData = userRequest.data[0]
```

Reply:
<img width="311" alt="image" src="https://user-images.githubusercontent.com/107163260/228872920-e9777631-7ef9-44f6-a735-db7b1846b0b5.png">

Reply:
I am using make API call in two place
1. https://github.com/Real-Dev-Squad/website-www/pull/432/files#diff-b8034978785404c0d5db1ed2aa00f41fecf7baaf67f71b348a0a1806a5b24017R255

<img width="637" alt="image" src="https://user-images.githubusercontent.com/107163260/230154399-dd480b7b-48f3-427f-9eb7-d908d020e9e3.png">

2.  https://github.com/Real-Dev-Squad/website-www/pull/432/files#diff-b8034978785404c0d5db1ed2aa00f41fecf7baaf67f71b348a0a1806a5b24017R291

<img width="335" alt="image" src="https://user-images.githubusercontent.com/107163260/230155338-b0678de1-34e5-4b9e-87fa-9ae09ad4214c.png">


and you said to do this
check this @shmbajaj you write on the time of review...if i do this then i focus on particular api call..Is I am right??
 https://github.com/Real-Dev-Squad/website-www/pull/432#discussion_r1153365771

Reply:
Talked to @shmbajaj and i am right...resolved